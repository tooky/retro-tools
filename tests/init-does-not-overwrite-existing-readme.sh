mkdir -p doc/retro
echo "A different readme" > doc/retro/README.md
retro init
ls doc/retro
cat doc/retro/README.md
