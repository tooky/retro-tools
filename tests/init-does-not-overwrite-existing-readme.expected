mkdir -p doc/retro
echo "A different readme" > doc/retro/README.md
retro init
You've already set up a retro folder!
ls doc/retro
README.md
cat doc/retro/README.md
A different readme
